bin/oneoff/data_fix/fix_invalid_users

```diff
@@ -74,17 +74,17 @@ users_destroyed = 0
 destroy_user_errors = 0
 User.where(provider: nil).
   find_in_batches(start: START_USER_ID, finish: END_USER_ID, batch_size: BATCH_SIZE) do |user_batch|
-  ActiveRecord::Base.transaction do
-    user_batch.each do |user|
+  user_batch.each do |user|
+    ActiveRecord::Base.transaction do
       users_destroyed += 1
       CDO.log.info "Destroying user: id=#{user.id}"
       user.destroy
-    rescue Exception => e
-      destroy_user_errors += 1
-      CDO.log.error e
-    end
 
-    raise ActiveRecord::Rollback unless options[:actually_update]
+      raise ActiveRecord::Rollback unless options[:actually_update]
+    end
+  rescue Exception => e
```

Comment:
Rescuing Exception [can cause unanticipated behavior](https://thoughtbot.com/blog/rescue-standarderror-not-exception).
```suggestion
  rescue StandardError => e
```